internal/controller/storageclient_controller.go

```diff
@@ -76,6 +76,8 @@ const (
 	vgscClusterIDIndexName = "index:volumeGroupSnapshotContentCSIDriver"
 
 	VolumeGroupSnapshotClassCrdName = "volumegroupsnapshotclasses.groupsnapshot.storage.k8s.io"
+
+	storageClaimFinalizer = "storageclaim.ocs.openshift.io"
```

Comment:
I don't think there is a point in having this here as a constant where the entire code is going away on the next release. I would prefer to just have everything local to a single function.